.regal/config.yaml

```diff
@@ -4,3 +4,7 @@ rules:
   style:
     todo-comment:
       level: warning
+    file-length:
+      level: warning
+    line-length:
+      max-line-length: 130
```

Comment:
This conflicted with `make `fmt`, so increased it.

Reply:
ack. I can't tell where `opa fmt` sets this to 130 though.

Nit: it would be nice to have a separate commit for this.